libyasm/tests/utf.asm

```diff
@@ -0,0 +1,6 @@
+dw __utf16__("hi Ты"),0
+dw __utf16le__("hi Ты"),0
+dw __utf16be__("hi Ты"),0
+dd __utf32__("hi Ты"),0
+dd __utf32le__("hi Ты"),0
+dd __utf32be__("hi Ты"),0
```

Comment:
none of these characters are >0xFFFF and so surrogate pair utf16 encoding is not tested here.